operator/main.py

```diff
@@ -315,16 +314,17 @@ def deploy_storage_class():
     logging.info(logf("Deployed StorageClass", manifest=filename))
 
 def send_analytics_tracker(ga_id):
```

Comment:
If possible we can move this function to `$SRC/lib/kadalulib.py`

Reply:
Yes, thats a good idea. We would reduce the mistakes.

Reply:
Done